drivers/clock_control/Kconfig.nrf

```diff
@@ -47,14 +47,51 @@ config CLOCK_CONTROL_NRF_K32SRC_EXT_FULL_SWING
 
 endchoice
 
-config CLOCK_CONTROL_NRF_K32SRC_BLOCKING
-	bool "Blocking 32KHz crystal oscillator startup"
-	depends on CLOCK_CONTROL_NRF_K32SRC_XTAL
+config CLOCK_CONTROL_NRF_K32SRC_RC_CALIBRATION
+	bool
+	depends on !SOC_SERIES_NRF91X
+	default y if CLOCK_CONTROL_NRF_K32SRC_RC
+
+if CLOCK_CONTROL_NRF_K32SRC_RC_CALIBRATION
+
+config CLOCK_CONTROL_NRF_CALIBRATION_PERIOD
+	int "Calibration opportunity period (in 250ms units)"
+	default 1
```

Comment:
Why is the default 1?  Last I looked the recommended interval was 4 s to maintain accuracy required for Bluetooth.  (That was some time back, so maybe it does need to be done at 4 Hz now.)

Reply:
shouldn't then CLOCK_CONTROL_NRF_CALIBRATION_MAX_SKIP = 1? With 2 that would result in 12 seconds if no temp change.

Reply:
applied defaults as suggested (but set CLOCK_CONTROL_NRF_CALIBRATION_MAX_SKIP = 1 to achieve calibration every 8s when no temperature change).